website/content/en/highlights/2022-03-28-kube-for-kubernetes_logs.md

```diff
@@ -0,0 +1,31 @@
+---
+date: "2022-03-28"
+title: "`kube` for `kubernetes_logs`"
+description: "The `kubernetes_logs` source is now powered by the `kube-rs` library"
+authors: ["spencergilbert"]
+pr_numbers: [11714]
+release: "0.21.0"
+hide_on_release_notes: false
+badges:
+  type: enhancement,breaking change
```

Comment:
Do we support commas here for the badges? I'm not seeing any other highlights that have multiple. I guess I'd leave this as `breaking change` if nothing else.

Reply:
Ah, mixed that up with `domains`